src/components/button/utils.tsx

```diff
@@ -6,8 +6,10 @@ import { LoadingIcon } from '../loading';
 export type ButtonModes =
   | 'interactive_0'
   | 'interactive_0--bordered'
+  | 'interactive_0--bordered--customized_off_palette'
```

Comment:
Would something like `'interactive_0--bordered-light-outline'` suffice?

Reply:
Yes, that works too.